src/Magnum/BulletIntegration/Test/CMakeLists.txt

```diff
@@ -25,5 +25,5 @@
 #   DEALINGS IN THE SOFTWARE.
 #
 
-corrade_add_test(BulletIntegrationTest IntegrationTest.cpp LIBRARIES MagnumBulletIntegration ${BULLET_LIBRARIES})
+corrade_add_test(BulletIntegrationTest IntegrationTest.cpp LIBRARIES MagnumBulletIntegration)
```

Comment:
Ah. Okay. That makes my previous comment on this line obsolete :) Would it be possible to discard the change from the commits altogether? ;)